composer.json

```diff
@@ -69,11 +69,13 @@
     "drupal/config_inspector": "^1",
     "drupal/devel": "^4",
     "drupal/drupal-extension": "^4",
-    "drupal/upgrade_rector": "^1",
     "drupal/upgrade_status": "^3",
```

Comment:
Why the module removed? It helps to create patches via rector

Reply:
it was removed after this discussion https://projects.skilld.cloud/skilld/nes-clv-coupons/-/merge_requests/2164#note_724502

and we're using rector directly for SDC tests

Reply:
It needs research because rector got a lot of facelift last 3 weeks, at least 50% deprecations from 9 to 10 already automatically covered by new rector

Reply:
@davidferlay https://twitter.com/nmdmatt/status/1427737526903746564?s=20

Reply:
it is not rector removal, it is removal of drupal module only

rector itself was upgraded to latest version
`"palantirnet/drupal-rector": "^0.11.2",`